src/main/java/ServiceLayer/UserService.java

```diff
@@ -1,57 +1,62 @@
 package ServiceLayer;
 
 import java.util.logging.Logger;
+
+import org.springframework.stereotype.Service;
+
 import java.util.logging.Level;
 import Domain.UserController;
 
+@Service
 public class UserService {
     private UserController userController;
     private TokenService tokenService;
     private static final Logger logger = Logger.getLogger(UserController.class.getName());
 
-    public UserService(){
+    public UserService() {
         userController = new UserController();
         tokenService = new TokenService();
     }
 
-    public Response logIn(String userName, String password) throws Exception {
+    public Response logIn(String userName, String password) {
         Response response = new Response();
         try {
             userController.logIn(userName, password);
             logger.info("User logged in: " + userName);
-            response.setReturnValue(tokenService.generateToken(userName));
```

Comment:
keep that name pls. its better for functions.

Reply:
What name?